README.md

```diff
@@ -75,33 +77,33 @@ github "YusukeHosonuma/SwiftPrettyPrint"
 ```swift
 dependencies: [
     .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", from: "0.0.2"),
-],
+]
 ```
 
 or add from Xcode 10+.
 
-## Recommend Setting 📝
+## Recommend Settings 📝
 
 When you need debug, you don't want to write `import` statement.
```

Comment:
I think `When you need debug` is unnecessary, so I recommend the following text.
`When you don't want to write `import` statement.`

Reply:
Thanks for proposal, I've fixed ! ( ef548de )

Reply:
(perhaps, It may prefer `when` to `if`😅)

Reply:
fixed!! ( 87acef1 )